test/test_fitsblob.py

```diff
@@ -19,7 +19,8 @@ class PyfitsFitsImage(unittest.TestCase):
     def setUp(self):
         self.hudelist = fitsopen(FITS_FILE)
 
-    def test_tkp_open(self):
+    @staticmethod
```

Comment:
Why?  I'm not sure I follow.

Reply:
My linter complained that `test_tkp_open` was defined as an instance method, while there is no use of `self`. Makes sense.
With this adjustment, my linter stopped complaining.

Reply:
Strange, do you remember what was the error?  I don't think you are supposed to have tests as classmethods, otherwise setup and teardown won't work properly.  The test is a bit weird though, I would actually test (assert) something after opening the file.

We can address this later, for the moment I'll revert the staticmethod bit.